README.md

```diff
@@ -6,7 +6,69 @@
   <a href="https://circleci.com/gh/kids-first/kf-ui-data-tracker"><img src="https://img.shields.io/circleci/project/github/kids-first/kf-ui-data-tracker.svg?style=for-the-badge"></a>
 </p>
 
-Kids First Data Tracker
-=======================
+# Kids First Data Tracker
 
 Investigator study management and progress tracking.
+
+## Development Quick Start
+
+#### Setup backend API service
+
+If you haven't already, create the kf-data-stack network:
+
+```
+docker network create kf-data-stack
+```
+
+Checkout kf-api-study-creator master and bring up the study creator API:
+
+```
+docker-compose up
+```
+
+For details, go to https://github.com/kids-first/kf-api-study-creator
+
+#### Setup front-end server
+
+If you haven't already, checkout kf-ui-data-tracker master:
+
+```
+git clone https://github.com/kids-first/kf-api-study-creator.git
+cd kf-ui-data-tracker
+```
+
+Install all dependencies:
+
+```
+yarn
+```
+
+Start the server:
+
+```
+yarn start
+```
+
+#### Access kf-ui-data-tracker front-end
```

Comment:
We should probably have a section documenting the configurable variables in `src/common/global.js`.
These are important to make things work, otherwise users won't be able to log in or maybe they will need to change the api's url.